docs/script.js

```diff
@@ -1,27 +1,250 @@
+    document.querySelector('.post > p').style.lineHeight = '24px';
+    document.querySelector('.post > ul').style.marginTop = '34px';
+    document.querySelector('.post > ul').style.marginBottom = '10px';
+    document.querySelector('.post > h3').style.fontSize = '20px';
+  }
+}
+
+// Desplays Details of the card Project
+function displayDetails(event) {
+  const card = cardSelected(event);
+  // Non style related
+  createQuitBtn(document.querySelector('#section-4 article'));
+  // Create a links for buttons
+  const link = createAlink(document.querySelector('#section-4 article'));
+  const link2 = createAlink(document.querySelector('#section-4 article'));
+  // See Live button
+  const seeLiveBtn = createButton(link);
+  seeLiveBtn.textContent = 'See Live';
+  const icon = createIcon(seeLiveBtn);
+  icon.setAttribute('src', 'sprites/Icon.svg');
+  // See source button
+  const seeSourceBtn = createButton(link2);
+  seeSourceBtn.textContent = 'See Source';
+  const icon2 = createIcon(seeSourceBtn);
+  icon2.setAttribute('src', 'sprites/icon2.svg');
+  // Delete existing li from ul
+  document.querySelectorAll('#section-4 > article ul li').forEach((element) => {
+    element.classList.add('hidden');
+  });
+  // Style related changes
+  btnDetails[0].style.display = 'none';
+  contentDetails.forEach((e) => e.classList.add('hidden'));
+  document.getElementById('section-2').style.display = 'none';
+  document.getElementById('section-4').classList.add('blur-bg');
+  document.querySelector('#section-4 > article').classList.add('articleStyle');
+  header3.classList.add('headerMobile');
+  imgPreview.style.order = '2';
+  imgPreview.style.marginTop = '24px';
+  textPreview.classList.add('p-style');
+  listOfTechnologies.classList.add('deco');
+  listOfTechnologies.style.gridTemplateColumns = '1fr 1fr 1fr';
+  document.querySelector('#section-4 > article button').style.order = '5';
+  link.classList.add('link1');
+  link2.classList.add('link2');
+  seeLiveBtn.classList.add('but');
+  seeSourceBtn.classList.add('but');
+  seeLiveBtn.style.height = '48px';
+  seeSourceBtn.style.height = '48px';
+  seeLiveBtn.style.width = '108px';
+  seeSourceBtn.style.width = '124px';
+  // changes related to each individual card
+  switch (card) {
+    case 'post-1':
+      generatePost(0, link, link2);
+      break;
+    case 'post-2':
+      generatePost(1, link, link2);
+      break;
+    case 'post-3':
+      generatePost(2, link, link2);
+      break;
+    case 'post-4':
+      generatePost(3, link, link2);
+      break;
+    default:
+      generatePost(4, link, link2);
+  }
+  document.getElementById('stylesheet').disabled = true;
+  document.querySelector('#section-4 > article').appendChild(div);
+  const ele = document.querySelectorAll('#section-4 > article a');
+  ele.forEach((element) => {
+    div.appendChild(element);
+  });
+  div.classList.add('dov');
+  div.setAttribute('id', 'dov');
+  myFunction(x);
 }
```

Comment:
There is an issue with your images not loading in your popup. Your popup image is missing as also the icons for the buttons. Kindly make the necessary changes so that the issue is fixed 😉

![image](https://user-images.githubusercontent.com/15807343/162110049-22ee4f35-fa7b-4b0a-8cc5-eb9bcf18a7ad.png)